packages/playground/src/calculator/pricing_calculator.vue

```diff
@@ -226,8 +253,12 @@ export default {
       ipv4: false,
       useCurrentBalance: true,
     });
-
-    const tftPriceTask = useAsync(() => calculator.tftPrice(), { init: true, default: 0 });
+    const dedicatedUpgradePrice = ref(Math.ceil(114.473 * 18));
+    const sharedUpgradePrice = ref(Math.ceil(228.947 * 18));
```

Comment:
why do we use `ref` are the values will be changed at some point? if not, please update it to be only const

```suggestion
    const dedicatedUpgradePrice = Math.ceil(114.473 * 18);
    const sharedUpgradePrice = Math.ceil(228.947 * 18);
```